docs/source/workflows/predictor_evaluation_workflows.rst

```diff
@@ -0,0 +1,264 @@
+When defining an evaluator, the top-level metrics should be the union of all metrics computed across all responses.
+The evaluator will only compute the subset of metrics valid for each response.
+
+Cross-validation evaluator
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+A :class:`~citrine.informatics.predictor_evaluator.CrossValidationEvaluator` performs k-fold cross-validation on a predictor.
+In addition to a name, set of responses to validate and metrics to compute, this evaluator defines the number of folds, number of trials and set of descriptor keys to ignore when grouping.
+The latter allows candidates with different values for the specified (ignored) keys and identical values all other predictor inputs to be placed in the same fold.
+
+Cross-validation can only be evaluated on predictors that define training data.
+During cross-validation, the predictor's training data is partitioned into k equally sized folds.
+Each fold acts as the test set once, and the remaining k-1 folds are used as training data.
+When the number of folds equals the number of training data points, the analysis is equivalent to leave-one-out cross-validation.
+Metrics are computed by comparing the model's predictions to observed values.
+
+Predictor evaluation metrics
+----------------------------
+
+Predictor evaluation metrics are defined as part of a :class:`~citrine.informatics.predictor_evaluator.PredictorEvaluator`.
+Available metrics depend on response type.
+For numeric responses, the following metrics are available:
+
+  - *Root-mean squared error* (:class:`~citrine.informatics.predictor_evaluation_metrics.RMSE`): square root of the average of the squared prediction error.
+    RMSE is a useful and popular statistical metric for model quality.
+    RMSE is optimized by least-squares regression, and in that sense is the most "natural" measure for it; it has the same units as the predicted quantity, and corresponds to the standard deviation of the variance not explained by the predictor.
+    Lower RMSE means the model is more accurate.
+  - *Non-dimensional error* (:class:`~citrine.informatics.predictor_evaluation_metrics.NDME`): RMSE divided by the standard deviation of the observed values in the test set.
+    (If training and test set are drawn from the same distribution, the standard deviation of the test set observed values is equivalent to the RMSE of a model that always predicts the mean of the observed values).
+    NDME is a useful non-dimensional model quality metric.
+    A value of NDME == 0 is a perfect model.
+    If NDME == 1, then the model is uninformative.
+    An acceptable NDME depends on how the model is used.
+    Generally, NDME > 0.9 indicates a model with low accuracy.
+    If 0.9 > NDME > 0.6, this model is typically a good candidate for a design workflow.
+    Lower values of NDE indicate increasingly accurate models.
```

Comment:
This separation into NDME > 0.9, 0.6 < NDME < 0.9, and NDME < 0.6 has never made sense to me. Is a model with NDME below 0.6 _not_ a good candidate for a design workflow?

```
If NDME == 1, then the model is uninformative.
Generally, models with NDME < 0.9 can be used in a design workflow.
```

Reply:
👍